src/tasks.ts

```diff
@@ -90,46 +83,35 @@ const RTasksMeta: RTaskMeta[] = [
         definition: {
             type: TYPE,
             code: ['devtools::install()'],
-            args: ['--no-echo', '--no-restore']
+            options: defaultOptions
         },
         name: 'Install',
         group: vscode.TaskGroup.Build,
         problemMatchers: []
     }
 ];
 
-
-
-
-const asRTask = async function (
-    folder: vscode.WorkspaceFolder | vscode.TaskScope,
-    meta: RTaskMeta
-): Promise<vscode.Task> {
-
-    const Rterm = await getRterm();
-    const RtermArgs = zipRTermArguments(meta.definition.code, meta.definition.args ?? []);
-
+function asRTask(rPath: string, folder: vscode.WorkspaceFolder | vscode.TaskScope, info: RTaskInfo): vscode.Task {
+    const args = makeRArgs(info.definition.options ?? defaultOptions, info.definition.code);
```

Comment:
```
info.definition.options ?? defaultOptions
```

Urgh, I'm angry at myself for not thinking of this. I kept trying to insert the defaultOptions into the definition which would cause user-defined tasks to fail to resolve. Just inserting it here in the call to `makeRArgs` solves that problem -.-